cbrain_task_descriptors/ndmg-docker.json

```diff
@@ -1,236 +1,247 @@
+        }, 
+        {
+            "name": "Atlas image (MNI152)", 
+            "value-key": "[ATLAS]", 
+            "type": "String", 
+            "value-choices": [
+                "/ndmg_atlases/atlas/MNI152_T1_1mm.nii.gz"
+            ], 
+            "optional": false, 
+            "id": "atlas"
+        }, 
+        {
+            "name": "Atlas brain mask", 
+            "value-key": "[MASK]", 
+            "type": "String", 
+            "value-choices": [
+                "/ndmg_atlases/atlas/MNI152_T1_1mm_brain_mask.nii.gz"
+            ], 
+            "optional": false, 
+            "id": "mask"
+        }, 
+        {
+            "value-key": "[OUTDIR]", 
+            "optional": false, 
+            "type": "String", 
+            "id": "outdir", 
+            "name": "Output directory"
+        }, 
+        {
+            "name": "Desikan parcellation", 
+            "value-key": "[DESIKAN]", 
+            "type": "String", 
+            "value-choices": [
+                "/ndmg_atlases/labels/desikan.nii.gz"
+            ], 
+            "optional": true, 
+            "id": "desikan"
+        }, 
+        {
+            "name": "JHU parcellation", 
+            "value-key": "[JHU]", 
+            "type": "String", 
+            "value-choices": [
+                "/ndmg_atlases/labels/JHU.nii.gz"
+            ], 
+            "optional": true, 
+            "id": "jhu"
+        }, 
+        {
+            "name": "Talairach parcellation", 
+            "value-key": "[TALAIRACH]", 
+            "type": "String", 
+            "value-choices": [
+                "/ndmg_atlases/labels/Talairach.nii.gz"
+            ], 
+            "optional": true, 
+            "id": "talairach"
+        }, 
+        {
+            "name": "AAL parcellation", 
+            "value-key": "[AAL]", 
+            "type": "String", 
+            "value-choices": [
+                "/ndmg_atlases/labels/AAL.nii.gz"
+            ], 
+            "optional": true, 
+            "id": "aal"
+        }, 
+        {
+            "name": "Harvard-Oxford parcellation", 
+            "value-key": "[HARVARDOXFORD]", 
+            "type": "String", 
+            "value-choices": [
+                "/ndmg_atlases/labels/HarvardOxford.nii.gz"
+            ], 
+            "optional": true, 
+            "id": "harvardoxford"
+        }, 
+        {
+            "name": "CPAC200 parcellation", 
+            "value-key": "[CPAC200]", 
+            "type": "String", 
+            "value-choices": [
+                "/ndmg_atlases/labels/CPAC200.nii.gz"
+            ], 
+            "optional": true, 
+            "id": "cpac200"
+        }, 
+        {
+            "command-line-flag": "-c", 
+            "name": "Clean-up flag", 
+            "value-key": "[CLEAN]", 
+            "type": "Flag", 
+            "optional": true, 
+            "id": "clean"
+        }
+    ], 
+    "container-image": {
+        "index": "http://index.docker.io", 
```

Comment:
on my machine, this only worked when I removed the `http://` - can you please try on yours?
i.e. `docker pull http://docker.io/neurodata/ndmg:v0.0.50` vs `docker pull docker.io/neurodata/ndmg:v0.0.50`

Reply:
It failed on my side too with `invalid reference format`. The convert should be adapt then, no?

Reply:
Sure, though I'm not sure this is a converter-problem - we simply had an invalid parameter here for many descriptors, but CBRAIN ignored that parameter. Now that we're using it, we need it to make the parameter value correct.

Either way, I'll bake it into the converter to remove any leading "http*://" in the index, since that is invalid for docker

Reply:
I updated the importer, can you regen these files using the newest version of `bosh-import`? Thanks!